components/Chips/src/MDCChipCollectionViewCell.h

```diff
@@ -0,0 +1,28 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@class MDCChipView;
+
+/*
+ A collection view cell containing a single MDCChipView.
+ */
+@interface MDCChipCollectionViewCell : UICollectionViewCell
+
+@property(nonatomic, nonnull, strong, readonly) MDCChipView *chip;
```

Comment:
Probably better to just call this "chipView" so its type is clear (much like you wouldn't call an image view property "image")

Reply:
Fixed.